source/stylesheets/_tab-code_signing.scss.erb

```diff
@@ -1194,6 +1208,7 @@
 
 		@include media-below-mid {
 			display: block;
+			overflow-y: auto;
```

Comment:
This won't make `.content` scrollable, isn't that a problem?

Reply:
this made the content scrollable...

Reply:
I tested with Firefox 58.0.2 and did not worked. 
I suggest to use height if you have a flex element with overflow scroll.
for example `.vertical-scroll { height: inherit; }` and
`.popup .popup-body.snowflake article { max-height: calc(100% - $article-header-size); }` where the `$article-header-size = 36px;` as the header height - article padding `52 - 16`

Reply:
I also checked and the issue is that the `overflow-y: auto` only added for the small screen size(`@include media-below-mid`).
![screen shot 2018-02-13 at 14 58 33](https://user-images.githubusercontent.com/8809762/36154005-5b4a990a-10d0-11e8-910f-c4681552eabc.png)
![screen shot 2018-02-13 at 14 58 44](https://user-images.githubusercontent.com/8809762/36154006-5b678ab0-10d0-11e8-8536-edb3a9ddf8f1.png)


So, I added a class(`.vertical-scroll`) to the popup's article tag, which is doing the same thing now is everything is OK, but please test it again.